packages/core/src/Select/Select.tsx

```diff
@@ -72,6 +73,10 @@ export interface HvSelectProps<
   autoComplete?: string;
   /** Whether the width of the select panel can vary independently. */
   variableWidth?: boolean;
+  /** Size of the select button. */
+  size?: HvButtonProps["size"];
+  /** Variant of the select button. @default secondarySubtle */
+  variant?: HvButtonProps["variant"];
```

Comment:
We could `extend Pick<HvButtonProps, ...>` in all these button-like components so that we don't have to _"duplicate"_ the docs